Java_Lap trinh bat dong bo.txt

```diff
@@ -1 +1,2 @@
-Bất đồng bộ là ...
\ No newline at end of file
+Bất đồng bộ là ...
+Đồng bộ là trong thực hiện tuần tự từng tiến trình,  tiến trình này kết thức thì tiến trình khác mới được thực thi
```

Comment:
```suggestion
Đồng bộ là trong thực hiện tuần tự từng tiến trình,  tiến trình này kết thức thì tiến trình khác mới được thực thi
```

Reply:
Đồng bộ trong java (Synchronization in java) là khả năng kiểm soát truy cập của nhiều luồng đến bất kỳ nguồn tài nguyên chia sẻ (shared resource).